repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library.yaml

```diff
@@ -1,5 +1,66 @@
 ---
 /content/documents/administration/national-indicator-library:
+  /labels:
```

Comment:
Suggest moving to seperate file if you want it to make it onto the on demand environments, otherwise you'll have to create manually

Reply:
done